src/ticTacToeGame.java

```diff
@@ -1,11 +1,12 @@
 import java.awt.datatransfer.Clipboard;
 import java.lang.Math;
 import java.io.*;
+import java.util.Iterator;
 import java.util.Scanner;
 
 public class ticTacToeGame {
     /** 
-     * This class is made for a tic tac toe game, where two plyaers play and choose their own symbols.
```

Comment:
I think you should ask the user if he wants to resume the game or start a new one